Code/Tools/AssetProcessor/native/tests/assetmanager/AssetProcessorManagerTest.cpp

```diff
@@ -3625,18 +3625,18 @@ TEST_F(AssetProcessorManagerTest, SourceFileProcessFailure_AutoFailedLfsPointerF
     AZ::IO::FixedMaxPathString engineRoot, projectRoot;
     settingsRegistry->Get(engineRoot, AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
     settingsRegistry->Get(projectRoot, AZ::SettingsRegistryMergeUtils::FilePathKey_ProjectPath);
-    settingsRegistry->Set(AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder, m_tempDir.path().toUtf8().data());
-    settingsRegistry->Set(AZ::SettingsRegistryMergeUtils::FilePathKey_ProjectPath, m_tempDir.path().toUtf8().data());
+    settingsRegistry->Set(AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder, m_assetRootDir.path().toUtf8().data());
+    settingsRegistry->Set(AZ::SettingsRegistryMergeUtils::FilePathKey_ProjectPath, m_assetRootDir.path().toUtf8().data());
 
-    QDir tempPath(m_tempDir.path());
-    QString gitAttributesPath = tempPath.absoluteFilePath(".gitattributes");
+    QDir assetRootDir(m_assetRootDir.path());
+    QString gitAttributesPath = assetRootDir.absoluteFilePath(".gitattributes");
```

Comment:
Why is the test making a file in the real asset root dir? 
``` 
   m_assetRootDir = QDir(m_data->m_databaseLocationListener.GetAssetRootDir().c_str());
```

Doesn't this modify the environment for other tests?

Reply:
Discussed offline, asset root is still a mock for these tests

Reply:
Yeah, the asset root is a temporary directory for running automation tests and will be cleaned up after the tests are finished.